pkg/workloads/cortex/lib/packages.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2019 Cortex Labs, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from subprocess import run
+
+from cortex.lib.exceptions import UserException, CortexException
+
+
+def install(project_path):
```

Comment:
I spoke with Vishal, and we think it's probably safer to pip install from bash before running `python3 api.py`. Can you replace the entrypoint of the container to a new bash file (maybe `run.sh`) which runs the `pip install` and then starts `api.py`? You will have to pass through the args to `api.py` (you can use `"$@"`). Also, with this design it could make sense to unzip in the init container, or you could unzip in `run.sh`.